.gitmodules

```diff
@@ -74,6 +74,10 @@
     ignore = dirty
     path = third_party/onnx
     url = https://github.com/onnx/onnx.git
+[submodule "third_party/optimizer"]
```

Comment:
those code should not be here from https://github.com/pytorch/pytorch/pull/55889